dataflow/flex-templates/streaming_beam/streaming_beam_test.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import multiprocessing as mp
+import os
+import pytest
+import subprocess as sp
+import tempfile
+import time
+import uuid
+
+from google.cloud import bigquery
+from google.cloud import pubsub
+
+PROJECT = os.environ["GCLOUD_PROJECT"]
+UUID = str(uuid.uuid4()).split('-')[0]
+DATASET = 'beam_samples_{}'.format(UUID)
+TABLE = 'streaming_beam_sql'
+TOPIC = 'messages-{}'.format(UUID)
+SUBSCRIPTION = TOPIC
+
+
+@pytest.fixture
+def topic_path():
+    publisher_client = pubsub.PublisherClient()
+    topic_path = publisher_client.topic_path(PROJECT, TOPIC)
+    try:
+        publisher_client.delete_topic(topic_path)
+    except Exception:
+        pass
+    topic = publisher_client.create_topic(topic_path)
+    yield topic.name
+    publisher_client.delete_topic(topic_path)
+
+
+@pytest.fixture
+def subscription_path(topic_path):
+    subscriber = pubsub.SubscriberClient()
+    subscription_path = subscriber.subscription_path(PROJECT, SUBSCRIPTION)
+    try:
+        subscriber.delete_subscription(subscription_path)
+    except Exception:
+        pass
+    subscription = subscriber.create_subscription(subscription_path, topic_path)
+    yield subscription.name
+    subscriber.delete_subscription(subscription_path)
+
+
+@pytest.fixture
+def dataset():
+    bigquery_client = bigquery.Client(project=PROJECT)
+    dataset_id = '{}.{}'.format(PROJECT, DATASET)
+    dataset = bigquery.Dataset(dataset_id)
+    dataset = bigquery_client.create_dataset(dataset, exists_ok=True)
+    yield
```

Comment:
It would be nice to return the `project:dataset` string from here so that it can be used as `'--output_table', '{}.{}'.format(dataset, TABLE)` below.

Reply:
Done.